Dockerfile

```
@@ -1,42 +1,33 @@
 ARG base_image=ghcr.io/alphagov/govuk-ruby-base:3.3
 ARG builder_image=ghcr.io/alphagov/govuk-ruby-builder:3.3
 
-ARG google_package_keyring=/usr/share/keyrings/google-linux-signing.gpg
-
-
-FROM $builder_image AS builder
-
-WORKDIR /tmp
-ARG google_package_keyring
-ADD https://dl-ssl.google.com/linux/linux_signing_key.pub google.pub
-RUN gpg --dearmor < google.pub > "${google_package_keyring}"
+FROM --platform=$TARGETPLATFORM $builder_image AS builder
 
 WORKDIR $APP_HOME
 COPY Gemfile* .ruby-version ./
 RUN bundle install
 COPY . ./
 
-
-FROM $base_image
+FROM --platform=$TARGETPLATFORM $base_image
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Remove Cucumber advert.
 ENV CUCUMBER_PUBLISH_QUIET=true
 # Use the default glibc malloc.
 ENV LD_PRELOAD=""
 
-# Install Google Chrome and the corresponding version of ChromeDriver.
-ARG google_package_keyring
-
-COPY --from=builder $google_package_keyring $google_package_keyring
 RUN arch=$(dpkg --print-architecture) && \
-    echo "deb [arch=${arch} signed-by=${google_package_keyring}] https://dl.google.com/linux/chrome/deb/ stable main" \
-        > /etc/apt/sources.list.d/google.list
-RUN install_packages dumb-init google-chrome-stable unzip
-RUN npm install @puppeteer/browsers && \
-  npx @puppeteer/browsers install chromedriver@stable --path / && \
-  mv /chromedriver/*/chromedriver-linux64/chromedriver /usr/bin/chromedriver
-
+    apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 0E98404D386FA1D9 6ED0E7B82643E131 F8D2585B8783D481 \
+      54404762BBB6E853 BDE6D2B9216EC7A8 && \
+    echo "deb [arch=${arch}] http://deb.debian.org/debian trixie main" > /etc/apt/sources.list.d/debian.list && \ 
+    echo "deb [arch=${arch}] http://deb.debian.org/debian-security/ trixie-security main" >> /etc/apt/sources.list.d/debian.list && \
+    echo "deb [arch=${arch}] http://deb.debian.org/debian trixie-updates main" >> /etc/apt/sources.list.d/debian.list
+RUN install_packages dumb-init unzip 
+
+# Copy Apt Pinning to Container.
+COPY scripts/chromium.pref /etc/apt/preferences.d/chromium.pref
+# RUN apt-get -y remove chromium-chromedriver
```

couple of redundant comments
```suggestion
COPY scripts/chromium.pref /etc/apt/preferences.d/chromium.pref
```